Shared/Samples/Find closest facility to multiple points/FindClosestFacilityToMultiplePointsView.swift

```diff
@@ -0,0 +1,180 @@
+        let closestFacilityTask = ClosestFacilityTask(url: .sanDiegoNetworkAnalysis)
+        
+        /// The parameters to be passed to the task to find the closest facility.
+        var closestFacilityParameters: ClosestFacilityParameters?
+        
+        /// A list of facilities around San Diego, CA, USA.
+        let facilities = {
+            let facilityPoints = [
+                Point(x: -13_042_130, y: 3_860_128, spatialReference: .webMercator),
+                Point(x: -13_042_193, y: 3_862_449, spatialReference: .webMercator),
+                Point(x: -13_046_883, y: 3_862_705, spatialReference: .webMercator),
+                Point(x: -13_040_540, y: 3_862_925, spatialReference: .webMercator),
+                Point(x: -13_042_571, y: 3_858_982, spatialReference: .webMercator),
+                Point(x: -13_039_785, y: 3_856_693, spatialReference: .webMercator),
+                Point(x: -13_049_024, y: 3_861_994, spatialReference: .webMercator)
+            ]
+            return facilityPoints.map { Facility(point: $0) }
+        }()
+        
+        /// The graphic for the route.
+        private let routeGraphic = Graphic(
+            symbol: SimpleLineSymbol(style: .solid, color: .blue, width: 2.0)
+        )
+        
+        /// The graphic for the incident.
+        private let incidentGraphic = Graphic(
+            symbol: SimpleMarkerSymbol(style: .cross, color: .black, size: 20)
+        )
+        
+        init() {
+            // Add the incident graphics to the graphics overlay.
+            incidentGraphicsOverlay.addGraphics([routeGraphic, incidentGraphic])
+            
+            // Create graphics for all the facilities and add them to the graphics overlay.
+            let facilitySymbol = PictureMarkerSymbol(url: .hospitalImage)
+            facilitySymbol.height = 30
+            facilitySymbol.width = 30
+            
+            let facilityGraphics = facilities.map {
+                Graphic(geometry: $0.geometry, symbol: facilitySymbol)
+            }
+            facilityGraphicsOverlay.addGraphics(facilityGraphics)
+        }
+        
+        /// Updates the incident to a given point and routes to the closest facility accordingly.
+        /// - Parameter mapPoint: The point on the map at which to add the incident.
+        func updateIncident(to mapPoint: Point) async throws {
+            // Update the incident graphic to the new point.
+            incidentGraphic.geometry = mapPoint
+            
+            // Update the parameters with the new incident.
+            let incident = Incident(point: mapPoint)
+            closestFacilityParameters?.setIncidents([incident])
+            
+            // Route to the closest facility to the incident.
+            try await routeToClosestFacility()
+        }
+        
+        /// Routes to the closest facility to the incident using the closest facility task.
+        private func routeToClosestFacility() async throws {
```

Comment:
Change the signature of this method to below and update the route graphic at its callsite.

```swift
private func routeToClosestFacility(
    using closestFacilityParameters: ClosestFacilityParameters
) async throws -> ClosestFacilityRoute?
```

A few benefits:

- Now you can unwrap `closestFacilityParameters` at its callsite for only once, i.e., `guard let closestFacilityParameters else { return }`
- The method has less [side effect](https://en.wikipedia.org/wiki/Pure_function)